server/errorHandler.test.ts

```diff
@@ -19,7 +19,7 @@ describe('GET 404', () => {
       .expect(404)
       .expect('Content-Type', /html/)
       .expect(res => {
-        expect(res.text).toContain('NotFoundError: Not found')
+        expect(res.text).toContain(`There's no page linked to that web address`)
```

Comment:
THe current custom error has this statement in case of 404 errors. Current version in dev - https://send-legal-mail-dev.prison.service.justice.gov.uk/404error